### DISCUSSION TITLE
Import- and Export-feed pointing to specific file?

### DISCUSSION THREAD

**Discussion by lennart380** [atrocore/atrocore#285] [General]
Hello
I'm brand new to AtroPIM, and got the system running yesterday.
But i have a question regarding where imported and exported files are located on disk and if this can be changed by config or command?
My goal is to have a schedule task running import from a specific file, made by my ERP. And likewise an specified export-file what kan be used by other system. 
Can this be done? or is there a simpler way to go?

**Comment by atrolex** [comment]
Free module enable you import the data from the file provided manually. All automations are possible with paid modules (not listed in our store yet). These enable to automate import and export (via scheduled jobs) from/to databases, URLs (also with authorization) and via HTTP Request (eg REST API).
Alternatively, you can also create you own module to realize the functionality you need.
